### DISCUSSION TITLE
volentmonkey 2.12.14版本使用过程中遇到的问题

### DISCUSSION THREAD

**Discussion by violenttamper** [violentmonkey/violentmonkey#1269] [Q&A] [answered]
在vivaldi浏览器上使用暴力猴扩展的时候，遇到某些脚本实现错误，比如有一个bilibili脚本，它显示图标不全。还有当装好暴力猴扩展，然后添加同步脚本后，页面就无法复制内容了。

**Comment by tophf** [comment, marked as answer]
I don't see anything we can do here. Post a new [issue](https://github.com/violentmonkey/violentmonkey/issues) with specific details to reproduce the problem.

**Reply by violenttamper** [reply]
我最后排查出问题了，是某个脚本的问题，关了就好了
